asv_bench/benchmarks/groupby.py

```diff
@@ -345,6 +345,26 @@ def time_groupby_dt_timegrouper_size(self):
 
 
 #----------------------------------------------------------------------
+# groupby.cummin and groupby.cummax # GH 15048
+
+class groupby_cummax_cummin(object):
+    goal_time = 0.2
+
+    def setup(self):
+        np.random.seed(1234)
+        self.G = 1000
+        self.N = 10000
+        self.df = pd.DataFrame({'A': np.random.randint(0, self.G, size=self.N),
+                                'B': np.random.randn(self.N)})
+
+    def time_cummin(self):
+        self.df.groupby('A').cummin()
+
+    def time_cummax(self):
+        self.df.groupby('A').cummax()
```

Comment:
Ah thanks for pointing that out. These benches were transferred directly from #15048, but they are essentially redundant compared to the existing benches: `df.groupby('key').['value'].cummax()` (old benchmark) vs `df.groupby('key').cummax()` (#15048)